### PR TITLE
Added the rest of the Spacenet Datasets

### PR DESCRIPTION
**Pull request by kbgg**
Still needs publications/links & tools for 1,3,4,5,6.

Also when going through this noticed that two AOIs are missing from the SpaceNet 5 catalog. They're in a different location on the S3 bucket so getting those added as well.

### CODE REVIEW

**Comment by kbgg** [comment]
Looking into it further and the two AOIs missing are only found in the test_public folder and they do not contain the labels, only the test imagery so I don't think those are public and should not be included in catalog since there are no labels. Will confirm with the SpaceNet team

File: datasets/spacenet_1.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    CollectionID: sn1_AOI_5_Khartoum
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+DataAtWork:
+  Publications:
+    - Title: 2nd SpaceNet Competition Winners Code Release
+      URL: "https://medium.com/the-downlinq/2nd-spacenet-competition-winners-code-release-c7473eea7c11"
+      AuthorName: David Lindenbaum
+      AuthorURL: https://medium.com/@david.lindenbaum
+  Tools & Applications:
```
Comment (HamedAlemo) [inline datasets/spacenet_1.yaml:40]: - Title: SpaceNet Challenge 1 Implementations
      URL: "https://github.com/SpaceNetChallenge/BuildingDetectors"

File: datasets/spacenet_3.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    CollectionID: sn3_AOI_5_Khartoum
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+DataAtWork:
+  Publications:
+    - Title: 2nd SpaceNet Competition Winners Code Release
+      URL: "https://medium.com/the-downlinq/2nd-spacenet-competition-winners-code-release-c7473eea7c11"
+      AuthorName: David Lindenbaum
+      AuthorURL: https://medium.com/@david.lindenbaum
+  Tools & Applications:
```
Comment (HamedAlemo) [inline datasets/spacenet_3.yaml:36]: - Title: SpaceNet Road Detection and Routing Challenge Solutions
      URL: "https://github.com/SpaceNetChallenge/RoadDetector"

File: datasets/spacenet_4.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    CollectionID: sn4_AOI_6_Atlanta
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+DataAtWork:
+  Publications:
+    - Title: 2nd SpaceNet Competition Winners Code Release
+      URL: "https://medium.com/the-downlinq/2nd-spacenet-competition-winners-code-release-c7473eea7c11"
+      AuthorName: David Lindenbaum
+      AuthorURL: https://medium.com/@david.lindenbaum
+  Tools & Applications:
```
Comment (HamedAlemo) [inline datasets/spacenet_4.yaml:25]: - Title: SpaceNet Challenge Round 4: Off-Nadir Buildings Competitor solutions
      URL: "https://github.com/SpaceNetChallenge/SpaceNet_Off_Nadir_Solutions"

File: datasets/spacenet_4.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  - segmentation
+  - worldview-3
+Citation: "SpaceNet on Amazon Web Services (AWS). “Datasets.” The SpaceNet Catalog.  Last modified April 30, 2018. Accessed on [Insert Date]. https://spacenetchallenge.github.io/datasets/datasetHomePage.html."
+Resources:
+  - Description: SpaceNet 4 Atlanta Chipped Training Dataset
+    CollectionID: sn4_AOI_6_Atlanta
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+DataAtWork:
+  Publications:
```
Comment (HamedAlemo) [inline datasets/spacenet_4.yaml:20]: - Title: The SpaceNet Challenge Off-Nadir Buildings: Introducing the winners
      URL: "https://medium.com/the-downlinq/the-spacenet-challenge-off-nadir-buildings-introducing-the-winners-b60f2b700266"
      AuthorName: Nick Weir
      AuthorURL: https://www.linkedin.com/in/niweir/

File: datasets/spacenet_5.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    CollectionID: sn5_AOI_8_Mumbai
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+DataAtWork:
+  Publications:
+    - Title: 2nd SpaceNet Competition Winners Code Release
+      URL: "https://medium.com/the-downlinq/2nd-spacenet-competition-winners-code-release-c7473eea7c11"
+      AuthorName: David Lindenbaum
+      AuthorURL: https://medium.com/@david.lindenbaum
+  Tools & Applications:
```
Comment (HamedAlemo) [inline datasets/spacenet_5.yaml:32]: - Title: SpaceNet Challenge Round 5: Road Networks and Optimized Routing Competitor solutions
      URL: "https://github.com/SpaceNetChallenge/SpaceNet_Optimized_Routing_Solutions"

File: datasets/spacenet_5.yaml [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  - Description: SpaceNet 5 Moscow Chipped Training Dataset
+    CollectionID: sn5_AOI_7_Moscow
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+  - Description: SpaceNet 5 Mumbai Chipped Training Dataset
+    CollectionID: sn5_AOI_8_Mumbai
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+DataAtWork:
+  Publications:
```
Comment (HamedAlemo) [inline datasets/spacenet_5.yaml:27]: - Title: SpaceNet 5 Winning Model Release: End of the Road
      URL: "https://medium.com/the-downlinq/spacenet-5-winning-model-release-end-of-the-road-fd02e00b826c"
      AuthorName: Adam Van Etten
      AuthorURL: https://www.linkedin.com/in/adamvanetten/

- Title: Announcing the Winners of the SpaceNet 5 Challenge
      URL: "https://medium.com/the-downlinq/announcing-the-winners-of-the-spacenet-5-challenge-5b4e4d92e105"
      AuthorName: Adam Van Etten
      AuthorURL: https://www.linkedin.com/in/adamvanetten/

File: datasets/spacenet_1.yaml [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+  - Description: SpaceNet 1 Shanghai Chipped Training Dataset
+    CollectionID: sn1_AOI_4_Shanghai
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+  - Description: SpaceNet 1 Khartoum Chipped Training Dataset
+    CollectionID: sn1_AOI_5_Khartoum
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+DataAtWork:
+  Publications:
```
Comment (HamedAlemo) [inline datasets/spacenet_1.yaml:35]: - Title: Getting Started With SpaceNet Data
      URL: "https://medium.com/the-downlinq/getting-started-with-spacenet-data-827fd2ec9f53"
      AuthorName: Adam Van Etten
      AuthorURL: https://www.linkedin.com/in/adamvanetten/

File: datasets/spacenet_3.yaml [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    CollectionID: sn3_AOI_4_Shanghai
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+  - Description: SpaceNet 3 Khartoum Chipped Training Dataset
+    CollectionID: sn3_AOI_5_Khartoum
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+DataAtWork:
+  Publications:
+    - Title: 2nd SpaceNet Competition Winners Code Release
```
Comment (HamedAlemo) [inline datasets/spacenet_3.yaml:32]: - Title: SpaceNet Roads Extraction and Routing Challenge Solutions are Released
      URL: "https://medium.com/the-downlinq/spacenet-roads-extraction-and-routing-challenge-solutions-are-released-48830c5184ef"
      AuthorName: David Lindenbaum
      AuthorURL: https://www.linkedin.com/in/david-l-a7678b15/

File: datasets/spacenet_2.yaml [hunk 9]
```diff
@@ -12,19 +12,19 @@ Tags:
...
-    CollectionID: sn_AOI_4_Shanghai
+    CollectionID: sn2_AOI_4_Shanghai
     Type: Labels, Source Imagery
     License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
   - Description: SpaceNet 2 Khartoum Chipped Training Dataset
-    CollectionID: sn_AOI_5_Khartoum
+    CollectionID: sn2_AOI_5_Khartoum
     Type: Labels, Source Imagery
     License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
 DataAtWork:
```
Comment (HamedAlemo) [inline datasets/spacenet_2.yaml:30]: update AuthorURL for David Lindenbaum to https://www.linkedin.com/in/david-l-a7678b15/

File: datasets/spacenet_6.yaml [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+  - sar
+  - worldview-2
+Citation: "SpaceNet on Amazon Web Services (AWS). “Datasets.” The SpaceNet Catalog.  Last modified April 30, 2018. Accessed on [Insert Date]. https://spacenetchallenge.github.io/datasets/datasetHomePage.html."
+Resources:
+  - Description: SpaceNet 6 Rotterdam Chipped Training Dataset
+    CollectionID: sn6_AOI_11_Rotterdaam
+    Type: Labels, Source Imagery
+    License: "[CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/)"
+DataAtWork:
+  Publications:
```
Comment (HamedAlemo) [inline datasets/spacenet_6.yaml:22]: - Title: SpaceNet 6: Announcing the Winners
      URL: "https://medium.com/the-downlinq/spacenet-6-announcing-the-winners-df817712b515"
      AuthorName: Jake Shermeyer
      AuthorURL: https://www.linkedin.com/in/shermeyer/
- Title: SpaceNet 6: Multi-Sensor All Weather Mapping Dataset
      URL: "https://arxiv.org/abs/2004.06500"
      AuthorName: Jake Shermeyer, et al.

File: datasets/spacenet_6.yaml [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+    - Title: 2nd SpaceNet Competition Winners Code Release
+      URL: "https://medium.com/the-downlinq/2nd-spacenet-competition-winners-code-release-c7473eea7c11"
+      AuthorName: David Lindenbaum
+      AuthorURL: https://medium.com/@david.lindenbaum
+  Tools & Applications:
+    - Title: SpaceNet Round 2 Challenge Implementations
+      URL: "https://github.com/SpaceNetChallenge/BuildingDetectors_Round2"
```
Comment (HamedAlemo) [inline datasets/spacenet_6.yaml:29]: remove

File: datasets/spacenet_5.yaml [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+The SpaceNet 5 challenge sought to build upon the advances from SpaceNet 3 and test challenge participants to automatically extract road networks and routing information from satellite imagery, along with travel time estimates along all roadways, thereby permitting true optimal routing.
+<br><br>
+The task of this challenge was to output a detailed graph structure with edges corresponding to roadways and nodes corresponding to intersections and end points, with estimates for route travel times on all detected edges. You can find a detailed description of CosmiQ Works’ algorithmic baseline on their blog at [The DownLinQ](https://medium.com/the-downlinq/inferring-route-travel-times-with-spacenet-7f55e1afdd6d).
+<br><br>
+SpaceNet open sourced new data sets for the following cities: Moscow, Russia; Mumbai, India; and San Juan, Puerto Rico. For the first time in SpaceNet history, the final submissions were tested on a mystery city dataset that was revealed and open sourced at the end of the Challenge."
+Documentation: https://spacenet.ai/sn5-challenge/
+Contact: https://spacenet.ai/contact-us/
+CreatedBy: "[SpaceNet LLC](https://spacenet.ai/)"
+Tags:
+  - road networks
```
Comment (HamedAlemo) [inline datasets/spacenet_5.yaml:13]: road network (no s) to be consistent.

File: datasets/spacenet_6.yaml [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+<br><br>
+In this challenge, the training dataset contained both SAR and EO imagery, however, the testing and scoring datasets contained only SAR data. Consequently, the EO data could be used for pre-processing the SAR data in some fashion, such as colorization, domain adaptation, or image translation, but cannot be used to directly map buildings. The dataset was structured to mimic real-world scenarios where historical EO data may be available, but concurrent EO collection with SAR is often not possible due to inconsistent orbits of the sensors, or cloud cover that will render the EO data unusable."
+Documentation: https://spacenet.ai/sn6-challenge/
+Contact: https://spacenet.ai/contact-us/
+CreatedBy: "[SpaceNet LLC](https://spacenet.ai/)"
+Tags:
```
Comment (HamedAlemo) [inline datasets/spacenet_6.yaml:10]: add -off-nadir

**Pull request merged**